### DISCUSSION TITLE
Aggiornamento del runtime di haaska

### DISCUSSION THREAD

**Discussion by Newreader** [indomus/forum#1704] [GENERALE - Discussioni generiche]
La funzione lambda haaska da me creata seguendo la guida InDomus del momento supportava la versione di runtime disponibile allora, ovvero la Python 3.6.
Poiché tale versione non sarà più supportata in AWS lambda sarebbe opportuno che migrassi verso la python 3.9 (come giustamente è riportato nella guida InDomus attuale).
A tal proposito chiedo a chi magari ha dovuto fare la stessa operazione la correttezza del procedimento:
Nella guida per gli sviluppatori della documentazione AWS è enucleata la procedura per modificare la versione del runtime, ovvero dopo aver selezionato la funzione da modificare (nel mio caso haaska) occorre spostarsi nella scheda “Codice” per poi in impostazioni runtime scegliere “Modifica” e selezionare la versione 3.9. Le altre due voci “Gestore” e "Architecture" vanno lasciate inalterate (rispettivamente haaska.event_handler e x86_64). Infine salvare.
Questo procedimento è sufficiente per aggiornare la versione del runtime e, quindi, poter continuare ad utilizzare senza problemi la funzione lambda haaska, dato che il Gestore e l'Architeture rimangono invariati rispetto alla originale creazione di haaska con il python 3.6?
Pongo questa domanda perché nella documentazione predetta è riportata la seguente dicitura “Se si aggiorna la configurazione della funzione per utilizzare un runtime diverso, è necessario fornire un nuovo codice funzione compatibile con il runtime e l'architettura”.
Grazie.

**Comment by snakuzzo** [comment]
Sì...ti basta solo cambiare quello e sei a posto.

**Reply by Newreader** [reply]
Grazie snakuzzo.
